operations/boot_orchestration/Create_a_Session_Template_to_Boot_Compute_Nodes_with_SBPS.md

```diff
@@ -0,0 +1,185 @@
+# Create a Session Template to Boot Compute Nodes with Scalable Boot Projection Service (SBPS)
+
+When [compute nodes](../../glossary.md#compute-node-cn) are booted, the [Scalable Boot Projection Service (SBPS)](../../glossary.md#scalable-boot-projection-service-sbps)
+projects the root file system \(`rootfs`\) over the network to the compute nodes.ø
+
+This page covers the appropriate contents for a BOS session template in order to use SBPS.
+
+- [Boot set `rootfs_provider` parameter](#boot-set-rootfs_provider-parameter)
+- [Boot set `rootfs_provider_passthrough` parameters](#boot-set-rootfs_provider_passthrough-parameters)
+  - [`<transport>`](#transport)
+  - [`<schema version>`](#schema-version)
+  - [`<IQN Domain>`](#iqn-domain)
+  - [`<DNS SRV record reference>`](#dns-service-srv-record-reference)
+  - [`<client discovery timeout in seconds>`](#client-discovery-timeout-in-seconds)
+  - [`<ramroot>`](#ramroot)
+  - [Example `rootfs_provider_passthrough`](#example-rootfs_provider_passthrough)
+- [`root=` kernel parameter](#root-kernel-parameter)
+- [Boot set S3 parameters](#boot-set-s3-parameters)
+- [Example session template input file](#example-session-template-input-file)
+- [Creating a BOS session using the new template](#creating-a-bos-session-using-the-new-template)
+
+The Scalable Boot Projection Service (SBPS) is an optional provider for the `rootfs` on compute nodes.
+
+## Boot set `rootfs_provider` parameter
+
+The following value needs to be set in the boot set of the session template in order to make SBPS the `rootfs` provider:
+
+- `"rootfs_provider":` Set to `"sbps"`
+
+## Boot set `rootfs_provider_passthrough` parameters
+
+For SBPS, the `rootfs_provider_passthrough` boot set parameter is customized according to the following format:
+
+```text
+rootfs_provider_passthrough=<transport>:<schema version>:<IQN Domain>:<DNS SRV record reference>:<client discovery timeout in seconds>
+```
+
+The following values need to be set in the boot set of the session template in order to make SBPS the `rootfs` provider:
+
+- `"rootfs_provider":` Set to `"sbps"`
+- To use the Node Management Network (NMN) for content projection,
+  - `"rootfs_provide_passthrough"`: Set to `"sbps:v1:iqn.2023-06.csm.iscsi:_sbps-nmn._tcp.odin.hpc.amslabs.hpecorp.net:300"`
+- To use the High Speed Network (HSN) for content projection,
+  - `"rootfs_provide_passthrough"`: Set to `"sbps:v1:iqn.2023-06.csm.iscsi:_sbps-hsn._tcp.odin.hpc.amslabs.hpecorp.net:300"`
+
+Note that `iqn.2023-06.csm.iscsi` is the IQN domain.
+
+The variables used in this parameter represent the following:
+
+### `<transport>`
+
+File system network transport. For example, `sbps` or `dvs`.
+
+Set to `sbps` to use SBPS. Can be left as an empty string to use the default value `dvs`.
```

Comment:
I'm a little confused about when I'd use `dvs` here? If I specify `dvs` is that not using the SBPS? If so, why is that in this document about creating a session template to boot computes with SBPS?